### PR TITLE
Remove legacy auth code

### PR DESCRIPTION
**Pull request by hbenl**

### CODE REVIEW

File: src/ui/utils/browser.ts [hunk 1]
```diff
@@ -1,15 +1,3 @@
-export function setUserInBrowserPrefs(user: { sub: string } | null) {
```
Comment (hbenl) [inline src/ui/utils/browser.ts:1]: This isn't actually supported in our browser anymore (I get "no such channel" errors when using it) so I removed it completely.

File: pages/[id]/sourcemap/[sourceId].tsx [hunk 2]
```diff
@@ -7,13 +7,12 @@ import { client, initSocket } from "protocol/socket";
...
 import renderSourcemap from "third-party/sourcemap-visualizer/sourcemapVisualizer";
 import { UIStore } from "ui/actions";
-import { setAppMode } from "ui/actions/app";
+import { getAccessToken, setAppMode } from "ui/actions/app";
 import { getAccessibleRecording } from "ui/actions/session";
 import { ExpectedErrorModal } from "ui/components/Errors/ExpectedErrorModal";
 import LoadingScreen from "ui/components/shared/LoadingScreen";
 import { useGetRecordingId } from "ui/hooks/recordings";
 import { useAppDispatch, useAppStore } from "ui/setup/hooks";
-import tokenManager from "ui/utils/tokenManager";
```
Comment (bvaughn) [inline pages/[id]/sourcemap/[sourceId].tsx:16]: Good riddens!

File: pages/_app.tsx [hunk 3]
```diff
@@ -131,18 +92,31 @@ const App = ({ apiKey, ...props }: AppProps & AuthProps) => {
...
       setToken({ token });
     }
+
+    async function getTokenFromBrowser() {
+      const token = await Promise.race([
+        new Promise<string>(resolve => listenForAccessToken(resolve)),
+        // if the user is not logged in, the listenForAccessToken callback
+        // is never called, so we add this timeout to continue without a token
+        new Promise<null>(resolve => setTimeout(() => resolve(null), 500)),
+      ]);
```
Comment (bvaughn) [inline pages/_app.tsx:111]: I _guess_ this makes sense but it seems kind of iffy. Is 500ms enough time to always be safe?

**Review by bvaughn — APPROVED** [review]

File: pages/_app.tsx (hunk 3 above)
Comment (hbenl) [reply pages/_app.tsx:111]: Yeah, I'd prefer if the browser sent us a "no access token" response. Anyway in my tests the browser responded within a few milliseconds, so 500ms should be enough.

**Pull request merged**